contract/ignition/modules/BankBeaconModule.ts

```diff
@@ -1,12 +1,8 @@
 import { buildModule } from '@nomicfoundation/hardhat-ignition/modules'
-import tipsModule from './ProxyModule'
 const bankBeaconModule = buildModule('BankBeaconModule', (m) => {
-  const { tips } = m.useModule(tipsModule)
   const beaconAdmin = m.getAccount(0)
   const bankImplementation = m.contract('Bank')
 
-  // Call the implementation function initialize by passing the address of the tips
-  m.call(bankImplementation, 'initialize', [tips.address])
```

Comment:
If the implementation is not initialized it can create security issue. 
So it's need to be initialized

Reply:
How so?

https://github.com/globe-and-citizen/cnc-portal/pull/359#discussion_r1778394117

Reply:
When an implementation contract is not initialized, anyone can call the initialize function with arbitrary parameters. This means an attacker can take control of critical variables, such as ownership, admin roles, or other sensitive state variables, which would normally be set during initialization. This allows them to control the contract or modify its behavior maliciously.

If the admin of an implementation call for exemple selfdestruct on an implementation, the beacon can't use this implementation anymore. 

That why the implementation need to be initialized. To avoid unexpected behaviour.

Reply:
Oh understood.
@farrellh1 please check this discussion